modules/streamloader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/streamloader/Resources.scala

```diff
@@ -188,31 +184,29 @@ object Resources {
   }
 
   /**
-    * Splits a List of As into two Lists, using the provided getSize function.
-    * Used to split a List[StreamLoaderRow] into a (remaining, toInsert) Lists.
+    * Batches a List of As into sub-Lists, using the provided getSize function.
+    * Used to split an unbounded List[StreamLoaderRow] into a smaller Lists
     * The max size of a Pub/Sub message is 10MB, so a lower threshold might result in events that get forever trapped in the stream.
     * @param batch A List of As (used with StreamLoaderRow)
     * @param getSize A function to get the size of a single A
     * @param threshold A limit to how big the toInsert List can be, as measured by toInsert.map(getSize).sum
```

Comment:
The idea of a `toInsert` List has been dropped; instead each List produced by this function should be able to be inserted. I suggest something like:

```
@param threshold A limit to how big each sub-List can be, as measured by subList.map(getSize).sum
```